modules/network/nsgs.tf

```diff
@@ -307,6 +294,17 @@ resource "oci_core_network_security_group_security_rule" "workers_healthcheck_in
 
 }
 
+resource "oci_core_network_security_group_security_rule" "workers_ingress_npn" {

```

Comment:
Since it's about pods communication, shouldn't this be covered under the pod NSG instead of the workers? The pods will be using the additional vnics and they should be covered by this rule: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/modules/network/locals.tf#L261

Reply:
The description comes from the documentation and could be more clear. e.g. `Pod -> NodePort` or `pod -> hostNetwork pod` would be permitted by this rule. I'll clarify in the description here.